### DISCUSSION TITLE
Cколько у нас активных юзеров?

### DISCUSSION THREAD

**Discussion by wrwrabbit** [wrwrabbit/Partisan-Telegram-Android#55] [Ideas]
@vivabelarus @ivanovivan12589 Как мы можем определить сколько у нас активных юзеров?

**Comment by vivabelarus** [comment]
Сколько человек использует приложение?

**Reply by wrwrabbit** [reply]
хз. не знаю можно ли как то определить

**Reply by vivabelarus** [reply]
Мы ж не собираем никакой информации. Поэтому никак, видимо

**Reply by ivanovivan12589** [reply]
Для этого вроде как или надо добавить сервис, который это трекает (но тогда у нас уже будет не телеграмм), или выложить куда-то, где можно смотреть количество скачиваний, и использовать это как приближение

**Reply by Korzhuck** [reply]
Не знаю актуально ли еще, но у вас же firebase подключен - он пишет эту инфу по умолчанию.

**Reply by cruszz** [reply]
Когда ты торгуешь веществами , МАСТ ХЭВ!

**Reply by cruszz** [reply]
Это шутка кто не понял)
